custom_components/myskoda/device_tracker.py

```diff
@@ -57,7 +57,10 @@ def latitude(self) -> float | None:  # noqa: D102
 
         self._update_device_from_coordinator()
 
-        return self.vehicle.position.positions[0].gps_coordinates.latitude
+        if len(self.vehicle.position.positions) > 0:
```

Comment:
nit: an non-empty list is True so you can just

```suggestion
        if self.vehicle.position.positions:
```